README.md

```diff
@@ -1,30 +1,84 @@
 
 # income-tax-subscription-self-employed-frontend
 
-This is a placeholder README.md for a new repository
+Scala/Play frontend web UI that provides screens for an existing SA Individual to provide the details required to sign-up a self-employed business.
 
 ### License
 
 This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
 
+. [Quick start](#Quick-start)
+- [Prerequisites](#Prerequisites)
+- [How to start](#How-to-start)
+- [How to use](#How-to-use)
+- [How to test](#How-to-test)
+2. [Persistence](#Persistence)
 
-1) **Start the service:**
+# Quick start
 
-   `sbt 'run 9563 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`
+## Prerequisites
 
+* [sbt](http://www.scala-sbt.org/)
+* MongoDB (*[See Persistence](#Persistence)*)
+* HMRC Service manager (*[Install Service-Manager](https://github.com/hmrc/service-manager/wiki/Install#install-service-manager)*)
+
+## How to start
+
+1) Run this service with
```

Comment:
It might be worth adding a start script: https://github.com/hmrc/income-tax-subscription-frontend/tree/main/scripts?